target_chains/solana/common_test_utils/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "common-test-utils"
```

Comment:
I extracted this here to be able to use it in `pyth-solana-receiver` and `pyth-push-oracle` at the same time